lottie/src/main/java/com/airbnb/lottie/Content.java

```diff
@@ -3,5 +3,7 @@
 import java.util.List;
 
 interface Content {
+  String getName();
```

Comment:
Is there a specific reason this needs to be on Content rather than DrawingContent other than consistency?